### PR TITLE
QA: FeaturedFoundersCarousel and FoundersImpactSlideshow

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Lots of minor design changes to these two carousels (they're very similar)
- Instead of sliding transition, the slides fade in
- Added "view next" right arrow button to navigate through the slides
- Added Heading to each carousel
- Fixes videos not appearing in `FeaturedFoundersCarousel` (shoutout @limsohee1002 😭 )
- Updates CTA link to be lilac-darkest

### Type(s) of changes

- [x] Bug fix
- [x] Update to an existing feature

### Motivation for PR

See description
### How Has This Been Tested?

View on homepage

**Previews:**
**FeaturedFoundersCarousel:**
https://share.getcloudapp.com/yAu6Z10Z

**FoundersImpacSlideshow:**
https://share.getcloudapp.com/Z4u6qA1W
### Follow-up PR

- Will probably need to be adjusted in QA

### CODE REVIEW

**Comment by chrismekim** [comment]
@limsohee1002 @mokaymokay FYI there will probably be styling adjustments after I get feedback in QA!

File: src/components/base/Video.tsx [hunk 1]
```diff
@@ -40,18 +40,16 @@ class Video extends Component<Props, State> {
       controls: true,
       loop: false,
     });
-
-    this.setState({ player });
```
Comment (mokaymokay) [inline src/components/base/Video.tsx:44]: why remove this? maybe remove it from state then?

File: src/components/layouts/FoundersImpactSlideshow.tsx [hunk 2]
```diff
@@ -126,30 +139,19 @@ class FoundersImpactSlideshow extends PureComponent<Props, State> {
                 ? generateCompanyDetailUrl(slide.company.companyDetailPageReference.slug)
                 : RouteMap.COMPANIES.path
             }
-            className="FoundersImpactSlideshow__right-tile bg-color-lilac-darker w100 text-left radius-xs flex flex-col mb6 md:mb0"
+            className="FoundersImpactSlideshow__right-tile bg-color-lilac-darker h100 w100 text-left radius-xs flex flex-col mb6md:mb0"
```
Comment (mokaymokay) [inline src/components/layouts/FoundersImpactSlideshow.tsx:142]: typo

**Review by mokaymokay — CHANGES_REQUESTED** [review]
doesn't seem like the video plays when you click play, might need to add an autoplay?

File: src/components/base/Video.tsx (hunk 1 above)
Comment (chrismekim) [reply src/components/base/Video.tsx]: oops, accidental removal! thanks!

**Review by mokaymokay — APPROVED** [review]

**Review by limsohee1002 — APPROVED** [review]

**Pull request merged**